images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -0,0 +1,201 @@
+function Get-AzCosmosDBEmulatorVersion {
+    $regKey = gci HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\* | gp | ? { $_.DisplayName -eq 'Azure Cosmos DB Emulator' }
+    $installDir = $regKey.InstallLocation
+    $exeFilePath = Join-Path $installDir 'CosmosDB.Emulator.exe'
+    $version = (Get-Item $exeFilePath).VersionInfo.FileVersion
+    return "Azure CosmosDb Emulator $version"
+}
+
+function Get-BazelVersion {
+    ((cmd /c "bazel --version 2>&1") | Out-String) -match "bazel (?<version>\d+\.\d+\.\d+)" | Out-Null
+    $bazelVersion = $Matches.Version
+    return "Bazel $bazelVersion"
+}
+
+function Get-BazeliskVersion {
+    ((cmd /c "bazelisk version 2>&1") | Out-String) -match "Bazelisk version: v(?<version>\d+\.\d+\.\d+)" | Out-Null
+    $bazeliskVersion = $Matches.Version
+    return "Bazelisk $bazeliskVersion"
+}
+
+function Get-CMakeVersion {
+    ($(cmake -version) | Out-String) -match  "cmake version (?<version>\d+\.\d+\.\d+)" | Out-Null
+    $cmakeVersion = $Matches.Version
+    return "CMake $cmakeVersion"
+}
+
+function Get-DockerVersion {
+    $dockerVersion = $(docker version --format "{{.Server.Version}}")
+    return "Docker $dockerVersion"
+}
+
+function Get-DockerComposeVersion {
+    $(docker-compose --version) -match "docker-compose version (?<version>\d+\.\d+\.\d+)" | Out-Null
+    $dockerComposeVersion = $Matches.Version
+    return "Docker-compose $dockerComposeVersion"
+}
+
+function Get-GitVersion {
+    $(git version) -match "git version (?<version>\d+\.\d+\.\d+)" | Out-Null
+    $gitVersion = $Matches.Version
+    return "Git $gitVersion"
+}
+
+function Get-GitLFSVersion {
+    $(git-lfs version) -match "git-lfs\/(?<version>\d+\.\d+\.\d+)" | Out-Null
+    $gitLfsVersion = $Matches.Version
+    return "Git LFS $gitLfsVersion"
+}
+
+function Get-InnoSetupVersion {
+    return $(choco list --local-only innosetup) | Select-String -Pattern "InnoSetup"
+}
+
+function Get-JQVersion {
+    $jqVersion = ($(jq --version) -Split "jq-")[1]
+    return "jq $jqVersion"
+}
+
+function Get-KubectlVersion {
+    $(kubectl version --client=true --short=true) -match "Client Version: v(?<version>.+)" | Out-Null
+    $kubectlVersion = $Matches.Version
+    return "Kubectl $kubectlVersion"
+}
+
+function Get-KindVersion {
+    $(kind version) -match "kind v(?<version>\d+\.\d+\.\d+)" | Out-Null
+    $kindVersion = $Matches.Version
+    return "Kind $kindVersion"
+}
+
+function Get-MinGWVersion {
+    (gcc --version | Select-String -Pattern "MinGW-W64 project") -match "(?<version>\d+\.\d+\.\d+)" | Out-Null
+    $mingwVersion = $Matches.Version
+    return "MinGW-w64 $mingwVersion"
```

Comment:
The project name is `Mingw-w64` - http://mingw-w64.org/doku.php / https://en.wikipedia.org/wiki/Mingw-w64

Reply:
Fixed, thanks!